packages/mobile/src/components/keyboard-avoiding/keyboard-avoiding.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC, ReactElement } from 'react';
+import { KeyboardAvoidingView } from 'react-native';
+import { PlatformOS } from '~/common/enums/enums';
+
+type Props = {
+  children: ReactElement;
+  style: Record<string, unknown>;
+};
+
+const KeyboardAvoiding: FC<Props> = ({ children, style }): ReactElement => {
+  return (
+    <KeyboardAvoidingView
+      style={style}
+      behavior={PlatformOS.ANDROID ? 'padding' : 'position'}
```

Comment:
`PlatformOS.ANDROID` here will always return `true` as it's just a string. I'd better introduce some constant instead:
```
// src/constants/constants.ts
//
import { Platform } from 'react-native';

const isAndroid = Platform.OS === 'android';
const isIOS = Platform.OS === 'ios';

export  { isAndroid, isIOS };

```